src/script.js

```diff
@@ -1,26 +1,14 @@
- function convertGrossToNet() {  
-    // Lấy giá trị thu nhập từ input  
-    const incomeInput = document.getElementById('incomeInput').value; 
-     
-    // Lấy phần tử bảng cho các ô hiển thị kết quả (Hàng thứ 2, cột thứ 1)
-    const grossCell = document.querySelector('table tr:nth-child(2) td:nth-child(1)');  
-    
-    // Cập nhật giá trị Lương Gross trong bảng  
-    grossCell.textContent = incomeInput || 0; // Nếu không có giá trị, hiển thị 0  
-}  
+import grossToNet from "./salary-app/grossToNet.js";
+import Big from "./lib/big.js";
+const A = Big(0.1).plus(Big(0.2)).toNumber();
+console.log(A);
 
-// Ngăn không cho form tự động gửi và tải lại trang khi nút được nhấn  
-function preventFormSubmission(event) {  
-    event.preventDefault();  
-}  
+document.querySelector(".btn-primary").addEventListener("click", async function () {
```

Comment:
The selector `.btn-primary` is not a good choice! We should use the id instead of css class name